app/routes/task_routes.py

```diff
@@ -63,76 +61,56 @@ def get_all_tasks():
 
     result_list =[task.to_dict()for task in tasks]
         
-    return jsonify(result_list or []),200
-
+    return result_list or []
 
 @bp.get("/<id>")
-
 def get_one_task(id):
     
     task =validate_model(Task,id)
     return task.to_dict()
 
-
 @bp.put("/<id>")
 def replace_task(id):
     task = validate_model(Task, id)
-    
     request_body = request.get_json()
     
     # if request_body["is_complete"]:
     #     task.completed_at = datetime.utcnow()
     # else:
     #     task.completed_at = None
     task.title = request_body["title"]
-    
     task.description = request_body["description"]
-    
     db.session.commit()
-    
     return Response(status = 204,mimetype ="application/json")
```

Comment:
Small spacing issues. 

We don't include spaces around the `=` when we are filling out a keyword argument in a function invocation.
- This differs from when it is used for variable assignment or as part of a comparison operator where we always surround `=` in spaces. 

```suggestion
    return Response(status=204, mimetype="application/json")
```